pubmedRetraction.html

```diff
@@ -0,0 +1,18 @@
+meta(charset='utf-8')
+html
+  head
+    link(rel='stylesheet', href='PMR.css')
+  body
+    script(src='//d3js.org/d3.v3.min.js')
```

Comment:
I think you want `http://...` here.

That said, it might be better to use `bower_components` to install and use d3 as opposed to fetching it from the web every time. You can do that by using `npm install -g bower` and `bower install d3` and then specifying it in the script tag as `bower_components/d3/lib/d3.v3.min.js` (double check this path, of course).